public/docs/ts/latest/guide/index.jade

```diff
@@ -1,109 +1,111 @@
+
+  The guide consists of chapters devoted to the challenges of building an application and
+  how to use Angular to solve problems.
+
+  A few early chapters are written as tutorials and are clearly marked as such.
   Most chapters are *not* tutorials and do not explain every step necessary to build the accompanying sample.
-  These chapters highlight key points in code but generally don't include the entire source.
-  We can get that source by way of the live link to the chapter's code sample.
-  
+  These chapters highlight key points in code but generally focus only on the pieces relevant to the chapter topic.
+
   ## Code Samples
-  
-  Every chapter includes code snippets ... snippets we can reuse in our own applications. 
-  Typically, these snippets are excerpts from a running sample application that accompanies the chapter.
-  We usually find a link to a live version of that sample near the top of the chapter ... like this one.
-  
+
+  Every chapter includes some code samples. These snippets are standalone snippets to highlight a particular concept and
+  excerpts from a running sample application that accompanies the chapter. Most chapters have an example that illustrates
+  the concepts found in the chapter. In those cases, you can find a link to a live version of that sample near the top of
+  the chapter that looks like this example that shows a basic Angular app.
+
   [Live Example](/resources/live-examples/architecture/ts/plnkr.html)
-  
+
   This link opens in a browser and runs the sample code supporting this chapter's architecture overview.
-  We can inspect, modify, save, and download the code.
-  
+  They use Plunkr, a service which allows you to live view, modify, save, and download the code.
+
   ## Learning Path
-  We don't have to read this guide straight through.  Most chapters stand on their own. 
-  We can browse to any of them as our interest or some necessity moves us.
-  
-  But there is a learning path for those of us who are new to Angular.
-
-  1. Read the [Architecture Overview](architecture.html) to get the big picture.
-  
-  1. Step *outside* the guide and try the [QuickStart](../quickstart.html). The QuickStart is the "Hello, World" of Angular 2. 
-  It shows us how to setup the libraries and tools we'll need to write *any* Angular app. 
-  
-  1. Follow the [Tutorial](../tutorial). The "Tour of Heroes" tutorial takes us step-by-step from where QuickStart leaves off 
-  to a simple data-driven app. 
-  Simple, yes, but with the essential characteristics we'd expect of a professional application:
-  a sensible project structure, data binding, master/detail, services, dependency injection, navigation, and remote data access. 
-  
-    The final iteration of the "[Tour of Heroes](#toh)" is a positive answer to that most important question: 
+  This guide can be read straight through, or used more like a reference. However, there is a recommended learning path for those who are new to Angular.
```

Comment:
The exact opposite of what I'm trying to say. I want the reader to know that they do NOT have to read it straight through and probably won't.